main.js

```diff
@@ -1,105 +1,81 @@
+/* global google */
 let map;
+// eslint-disable-next-line no-unused-vars
 let marker;

```

Comment:
da li ti ova marker varijabla uopste treba? nisam skontao gde se koristi?

Reply:
Ona se pojavljuje na 36 i 74 liniji. Pokusao sam i bez nje, pa dole samo new google.maps.Marker({...   , mapa u tom slucaju radi, ali eslint prijavljuje - Do not use 'new' for side effects. Resicu nekako. Sada vidim da ovde nista ne radi(valjda je do servera)??